traffic_ops/app/conf/cdn.conf

```diff
@@ -11,6 +11,11 @@
 		workers  => 12,     # Number of workers to run in parallel. Start with a small number.  Increase as need arises.
 	},
 	traffic_ops_golang_port => '443',
+	traffic_ops_golang_proxy_timeout => 60,
```

Comment:
would it be better to create a hash for these?  e.g.
```
traffic_ops_golang => {
   port => '443',
   proxy_timeout => 60,
...
}
```

Reply:
@dangogh My goal is to convert cdn.conf and create a new section for `traffic_ops_golang` that all these configs fall under then the traffic_ops_golang.config can just go away (for the moment)

Reply:
And creating a hash would mean a change to the `perlhash.go` file